clusters/app.ci/ci-grafana/datasources.yaml

```diff
@@ -18,6 +18,17 @@ spec:
       httpHeaderValue1: 'Bearer ${PROMETHEUS_USER_WORKLOAD_TOKEN}'
     type: prometheus
     url: https://thanos-querier-openshift-monitoring.apps.ci.l2s4.p1.openshiftapps.com
+  - name: prometheus-k8s
+    access: proxy
+    editable: false
+    jsonData:
+      httpHeaderName1: 'Authorization'
+      timeInterval: 5s
+      tlsSkipVerify: true
+    secureJsonData:
+      httpHeaderValue1: 'Bearer ${OPENSHIFT_MONITORING_CI_TOKEN}'
+    type: prometheus
+    url: https://thanos-querier-openshift-monitoring.apps.ci.l2s4.p1.openshiftapps.com
```

Comment:
can we use an internal URL instead?

Reply:
It is a grey area.
We do not know what is the best practice because it (cluster-monitoring) is not documented.
Because this way works on hive, let us keep the consistency for now.
One day, we will do it according to the doc.

Also notice that it is suggested by the user-workload to use the same URL (thanos's route's host).
I am not sure why it is not suggested to use the svc DNS.
So the hostname is semi-formally supported.

Reply:
I would prefer to use internal domains for any in-cluster communications (`hive` is out-of-cluster communication) to avoid the traffic going through load balancer. We can confirm with `openshift-monitoring`.

Reply:
How to use the internal svc is not documented (how to use the route's host is documented).
I will give it a try after merging and switch it if it works.